### PR TITLE
fixed config examples

### PR DESCRIPTION
**Pull request by fdistorted**

### CODE REVIEW

File: client/config.go [hunk 1]
```diff
@@ -3,12 +3,12 @@ package client
...
 
 type Config struct {
-	Subscriptions []string `yaml:"subscriptions"`
+	Subscriptions []string `hcl:"subscriptions"`
 }
 
 func (c Config) Example() string {
 	return `configuration {
 		//  Optional. if you not specified, cloudquery tries to access all subscriptions available to tenant
-		subscriptions = ["<YOU_SUBSCRIPTION_ID_HERE">]
```
Comment (irmatov) [inline client/config.go:12]: YOUR (and in three other places)

Comment (fdistorted) [reply client/config.go:12]: this is a removed line
my line looks next `subscriptions = ["<YOU_SUBSCRIPTION_ID_HERE>"]`

Comment (fdistorted) [reply client/config.go:12]: this is a removed line
my line looks next: `subscriptions = ["<YOU_SUBSCRIPTION_ID_HERE>"]`

**Review by roneli — APPROVED** [review]
LGTM

File: client/config.go (hunk 1 above)
Comment (irmatov) [reply client/config.go:12]: I know, you just swapped " and > characters. I mean while we are at it, we could also fix spelling. The intent here is "youR subscription id".

**Review by irmatov — APPROVED** [review]

**Pull request merged**